build.py

```diff
@@ -696,13 +714,15 @@ def create_dockerfile_linux(ddir, dockerfile_name, argmap, backends, repoagents,
 COPY --chown=1000:1000 --from=tritonserver_build /tmp/tritonbuild/install/repoagents repoagents
 '''
 
+    if target_platform() != 'ubuntu/arm64':
```

Comment:
I will note here the docker build breaks on a non-cuda x86 machine (where you would pass the target platform "ubuntu") with error: /bin/bash: ${{_CUDA_COMPAT_PATH}}/lib.real: bad substitution. I guess the assumption is that on x86 platforms you must have the nvidia container runtime.